plugins/tableselection/plugin.js

```diff
@@ -262,11 +262,15 @@
 			selection = editor.getSelection( 1 ),
 			selectedTable = getFakeSelectedTable( editor ),
 			target = evt.data.getTarget(),
-			cell = target && target.getAscendant( { td: 1, th: 1 }, true ),
-			table = target && target.getAscendant( 'table', true ),
 			tableElements = { table: 1, thead: 1, tbody: 1, tfoot: 1, tr: 1, td: 1, th: 1 },
 			canClear;
 
+		// Prevent of throwing error in console if target is undefined (#515).
+		if ( target.$.tagName !== undefined ) {
+			var cell = target && target.getAscendant( { td: 1, th: 1 }, true ),
+			table = target && target.getAscendant( 'table', true );
+		}
+
```

Comment:
You can check if object is wrapped on HTMLElement either by checking if [getName method](http://docs.ckeditor.com/#!/api/CKEDITOR.dom.element-method-getName) exists or by checking [type property](http://docs.ckeditor.com/#!/api/CKEDITOR.dom.element-property-type).

Here it would be simpler to leave the original logic untouched, but just to do an "early return", by adding

```
if( !evt.data.getTarget().getName ) {
	return;
}
```

At the beginning of the function.

That will be safer way of fixing it. So please revert the changes in this file, and just do an early return.